File: RailgunVB/Commands/Ping.vb
Imports Discord
Imports Discord.Commands
Imports Discord.WebSocket

Namespace Commands
    
    <Group("ping")>
    Public Class Ping
        Inherits ModuleBase
        
        Private ReadOnly _client As DiscordShardedClient
        
        Public Sub New(client As DiscordShardedClient)
            _client = client
        End Sub
    
        <Command>
        Public Function PingAsync() As Task
            Return ReplyAsync($"Discord Client Latency : {Format.Bold(_client.Latency.ToString())}ms")
        End Function
        
    End Class
    
End NameSpace

File: Railgun/Commands/Utilities/Ping.cs
using System.Threading.Tasks;
using Discord;
using Discord.WebSocket;
using Finite.Commands;
using Railgun.Core.Commands;

namespace Railgun.Commands.Utilities
{
    [Alias("ping")]
    public class Ping : SystemBase
    {
        private readonly DiscordShardedClient _client;

        public Ping(DiscordShardedClient client) => _client = client;

        [Command]
        public Task PingAsync()
            => Context.Channel.SendMessageAsync($"Discord Client Latency : {Format.Bold(_client.Latency.ToString())}ms")
    }
}